docs/source/quick_start/troubleshooting.rst

```diff
@@ -0,0 +1,85 @@
+
+Troubleshooting
+===============
+
+Have problems to make things work? Here are some common issues and their solutions
+
+MCDR won't start
+----------------
+
+* **mcdreforged: command not found**
+    * Try ``python -m mcdreforged``
+    * If you installed MCDR via pipx, ensure the pipx apps directory is in your PATH
+    * If you installed MCDR via pip globally, ensure the pip scripts directory is in your PATH
+    * If you installed MCDR via pip within venv, make sure you have activated the venv
+    * If you installed MCDR via system package manager... don't install MCDR via system package manager
+
+* **There're some error raised by Python**
+    * Install MCDR correctly. Don't start MCDR from source code
+    * Try to re-install MCDR, see if the problem persists
+    * If you installed MCDR via venv, make sure you have activated the venv
+    * Use ``which mcdreforged`` (``where mcdreforged`` in Windows), see which executable is being called
+
+Get in touch with :ref:`our kind community <quick_start/next_steps:Community>`
+
+The server won't start
+----------------------
+
+* Check out your start command
+* Check if java is installed and added in PATH
+* Try to run the server without MCDR, using the same command you configured
+* If you're directing the start command to a batch or shellscript, try the actual command instead of the script
+
+Learn more in :ref:`configuration:start_command`
+
+
+Garbled text / UnicodeDecodeError
+---------------------------------
+
+Usually, this is caused by the console encoding / decoding
+
+MCDR use UTF-8 as default encoding and decoding, so try to use UTF-8 in everything related to the server. See :ref:`configuration:encoding, decoding`
+
+Commands not working
+--------------------
+
+- **All commands are not responding in game, but working in console**
+- **Some commands perform incorrectly for specific players**
+
+    MCDR handle commands by listening to the server's console output. Make sure you are using the correct :ref:`Server Handler <configuration:handler>`
+
+    Your server may unexpectedly not compatible with handlers:
+ 
+    Maybe your server software is unique that not supported by built-in handlers
+
+    Maybe your server output is different. For example, if you allow players that not following the Mojang naming rule (``[A-Za-z0-9_]{3,16}``), the handler will not recognize them
```

Comment:
Add some examples? The user might not be able to figure out which case it belongs to with such a large amount of pure text

```
[09:00:00] [Server thread/INFO]: <Steve.the.Warrior> Hello
```

```
[09:00:00] [Server thread/INFO]: <[Warrior]Steve> Hello
```